testsuite/features/build_validation/retail/proxy_container_branch_network.feature

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@proxy
+@private_net
+@containerized_server
+Feature: Prepare the containerized branch server for PXE booting
+  In order to use SUSE Manager for Retail solution in a containerized setup
+  As the system administrator
+  I prepare the branch network in containerized setup
+
+  Scenario: Activate the branch network on the proxy
+    When I connect the second interface of the proxy to the private network
+    And I restart all proxy containers to let them pick new network configuration
+    Then the "dhcp_dns" host should be present on private network
+    And name resolution should work on private network
+
+  Scenario: Show the overview page of the containerized proxy
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Create branch terminals group
+    When I follow the left menu "Systems > System Groups"
+    And I follow "Create Group"
+    And I enter "example" as "name"
+    And I enter "Terminal branch: example.org" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group example created." text
+    # TODO: Re-add this when we onboard the proxy again (no more "Foreign" system type):
+    # When I follow "Target Systems"
+    # And I check the "proxy" client
+    # And I click on "Add Systems"
+    # Then I should see a "1 systems were added to example server group." text
```

Comment:
I think we can remove the TODO and enable the code. The HEAD CI passed core and the proxy is onboarded as normal Minion again.